packages/happy-dom/src/css/declaration/css-parser/CSSStyleDeclarationCSSParser.ts

```diff
@@ -12,24 +12,19 @@ export default class CSSStyleDeclarationCSSParser {
 		cssText: string,
 		callback: (name: string, value: string, important: boolean) => void
 	): void {
-		const parts = cssText.split(';');
-
-		for (const part of parts) {
-			if (part) {
-				const [name, value]: string[] = part.trim().split(':');
-				if (value) {
-					const trimmedName = name.trim();
-					const trimmedValue = value.trim();
-					if (trimmedName && trimmedValue) {
-						const important = trimmedValue.endsWith(' !important');
-						const valueWithoutImportant = trimmedValue.replace(' !important', '');
-
-						if (valueWithoutImportant) {
-							callback(trimmedName, valueWithoutImportant, important);
-						}
-					}
-				}
+		const rules = [
+			...cssText.matchAll(
```

Comment:
I believe that `Array.from()` is faster than "...".

Reply:
can't find benchmaks that prove that, only some references that in some engines spread operator can have limits regarding the maximum number of element to spread. I will change that.